linode/resource_linode_lke_cluster.go

```diff
@@ -406,6 +414,150 @@ func reconcileLKEClusterPoolSpecs(poolSpecs []linodeLKEClusterPoolSpec, pools []
 	return
 }
 
+func waitForClusterPoolReady(ctx context.Context, client *linodego.Client, errCh chan<- error, wg *sync.WaitGroup, clusterID, poolID int) {
+	eventTicker := time.NewTicker(500 * time.Millisecond)
```

Comment:
should this be configurable like Min/MaxRetryDelayMilliseconds

Reply:
good point, yeah this can feed off of a global configuration for polling.